src/DataHandling/Logs/DataDogAudit.php

```diff
@@ -16,6 +16,7 @@ class DataDogAudit extends AbstractBaseAudit
 
     const TEMP_KEY_READD = 'temp_readd';
     const TEMP_KEY_OLDVAL = 'temp_oldval';
+    const UNKNOWN_VERSION_CHANGE = 'unknown version';
```

Comment:
really small hint: maybe we can keep notation, that we separate words in const value with underscorce, like in TEMP_KEY_READD and TEMP_KEY_OLDVAL

Reply:
I wanted a key which does very likely not collision with a generated value. A key with a space in it will very likely not be generated.